translate_srt/__init__.py

```
@@ -0,0 +1 @@
+from .translate_srt import translate_srt, translate_text, main, SUPPORTED_LANGUAGES
```

Ruff (0.8.2)

`.translate_srt.translate_srt` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)